src/main/resources/assets/js/components/AppModalComponent.jsx

```diff
@@ -194,7 +219,7 @@ define([
               </StackedViewComponent>
             </TabPaneComponent>
             <TabPaneComponent
-              id="configuration"
+              id={"app"+model.get("id")+"/configuration"}
```

Comment:
Let's get some spaces in there: `id={"app" + model.get("id") + "/configuration"}`